app/src/main/java/com/rootstrap/android/ui/activity/main/targetpoint/MapFragment.kt

```diff
@@ -90,6 +100,48 @@ class MapFragment : PermissionFragment(), OnMapReadyCallback {
         setMapCamera(location)
     }
 
+    private fun addCircleMarkerForTarget(target: TargetModel) {
+        val position = LatLng(target.lat, target.lng)
+        target.topic?.run {
+            mMap.addMarker(
+                MarkerOptions()
+                    .position(position)
+                    .icon(bitmapDescriptorwithOvalBackground(requireContext(), getIconForTarget()))
+            )
+        }
+    }
+
+    private fun bitmapDescriptorwithOvalBackground(context: Context, @DrawableRes vectorDrawableResourceId: Int): BitmapDescriptor? {
```

Comment:
this line is too long, split this line
```suggestion
    private fun bitmapDescriptorwithOvalBackground(
        context: Context, 
        @DrawableRes vectorDrawableResourceId: Int
    ): BitmapDescriptor? {
```